### PR TITLE
Release

### PR DESCRIPTION
**Pull request by Kaaserne**
See pipeline `mlx90640_processor_c` cpp file for the references to the datasheet pages. Feedback from previous PR has been processed as well, including:
- Moved `data_extractor_s` to a namespace instead of class (the `_s` extension was faulty as well).
- Comment fix where `uint16_t` should be `int`
- `int` to `unsigned int` for indexing array values.

### CODE REVIEW

File: code/src/mlx90640_processor.cpp [hunk 1]
```diff
@@ -1,107 +1,152 @@
...
+                       &ksto,   &alpha_corr, &alpha_cp,  &cp_offset, &kv_cp,
+                       &kta_cp, &tgc,        &resolution};
+        // Sets all EEPROM data in mlx params
+        for (const auto &static_var : static_vars) {
+            static_var->extract();
+        }
 
-        Vdd25 = extract_data(registers::EE_VDD_PIX, 0x00FF, 0);
-        Vdd25 = (Vdd25 - 256) * 32 - 8192;
+        hwlib::cout << "Mlx90640 initialized!\n";
```
Comment (LRstudentHU) [inline code/src/mlx90640_processor.cpp]: Not required in master

**Review by LRstudentHU — COMMENTED** [review]
Very nice

File: code/src/mlx90640_processor.cpp (hunk 1 above)
Comment (MarcDirven) [reply code/src/mlx90640_processor.cpp]: Initializing takes some time, I think around 1/2 seconds approxemately, I thought maybe add a print statement to reduce confusion when other people continue to code on this module.

**Review by nicovanbentum — APPROVED** [review]
Much improved compared to the previous PR, couldn't find any notable issues.

**Review by LRstudentHU — APPROVED** [review]

**Pull request merged**